tests/testapps/CGCatalog/CGCatalog.vsimporter/CGCatalog-WinStore10/CGCatalog.vcxproj.filters

```diff
@@ -202,6 +202,8 @@
     <SBResourceCopy Include="..\..\CGCatalog\Resources\CloseSubPath.png" />
     <SBResourceCopy Include="..\..\CGCatalog\Resources\GetBoundingBox.png" />
     <SBResourceCopy Include="..\..\CGCatalog\Resources\CloseSubPath.png" />
+    <SBResourceCopy Include="..\..\CGCatalog\Resources\GetBoundingBox.png" />
+    <SBResourceCopy Include="..\..\CGCatalog\Resources\CloseSubPath.png" />
```

Comment:
I don't know if it supports wild cards, there seems to be a bigger problem overall though that I have no idea what's going on. Visual studio is adamant about creating multiple references to the same pictures... Just look at the few lines above. Removing these and re-adding the files to the project just results in these duplicates being recreated. Seems like an issue that needs more investigation.

---
In reply to: [91632444](https://github.com/Microsoft/WinObjC/pull/1507#discussion_r91632444) [](ancestors = 91632444)